client/components/RegionList.js

```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React, { useState, useEffect } from "react"
+import fetchData from "../functions/fetchData.js"
+
+const RegionList = (props) => {
+  const [regions, setRegions] = useState([])
+  const [selectedRegion, setSelectedRegion] = useState(null)
+
+  useEffect(() => {
+    fetchRegions
+  }, [])
+
+  const fetchRegions = async (id) => {
+    const data = await fetchData("/api/v1/pets")
+    setRegions(id)

```

Comment:
How is `data` being used in this function? It is returned from the fetch but the variable is never used